README.md

```diff
@@ -43,10 +43,12 @@ jobs:
   serial: true
   plan:
   - get: resource-web-app
+  - task: build
+    file: resource-web-app/build.yml
   - put: resource-deploy-web-app
     params:
-      manifest: resource-web-app/manifest.yml
-      path: resource-web-app
```

Comment:
Not the whole directory, just the manifest, which should just need the path to the built artifact, which would already be in the output.